### ISSUE TITLE
Package name, bundle identifier, source and many other get mixed up

### ISSUE THREAD

**Issue by eve0415** [zbrateam/Zebra#1667]
**Describe the bug**
The package name and bundle identifier, source are not showing correct information. There are somehow mixed up.

**To Reproduce**
Probably, when you first refreshing all the sources, Zebra crash (#1657) and this might be the reason.
(The database too save the wrong bundle identifier for package (name)

**Expected behavior**
Properly cache the data with right info and order.

**Device Information:**
 - iOS Version: 13.5
 - Zebra Version: beta7b
 - Device Type: iPhone
 - Jailbreak Type: unc0ver

**Additional context**
The package and the source are random.
Try removing `zebra.db` and reopen Zebra (+crash maybe) to have different package with different identifier.
If you need database (I saved it on my computer), I'll send it to you by DM

**Comment by wstyres** [comment]
Having the database and sources list would be very helpful, if you could send it over that would be great.

**Comment by wstyres** [comment]
I've just released version 1.2~beta7d. Please update to that version and give it a try. If the issue is still occurring. Please delete your database located at `/var/mobile/Library/Application Support/xyz.willy.zebra/zebra.db` and try opening Zebra again.

**Comment by eve0415** [comment]
Not having this problem anymore

**Comment by yur1xpp** [comment]
I'm still having this issue with AltDaemon package, where it got confused with purchased SwipeExpanderX package:
![image](https://user-images.githubusercontent.com/34767893/103399620-d23a5c00-4b7c-11eb-8b8d-efbf2b18031d.png)

**Comment by PoomSmart** [comment]
@yur1xpp What if you remove `/var/mobile/Library/Application Support/xyz.willy.zebra/zebra.db` and reopen Zebra?

**Comment by yur1xpp** [comment]
It’s the same unfortunately. I wanted to reset the sources cache too (not sure if it’s the same thing as removing the .db), but tapping reset doesn’t do anything.

**Comment by PoomSmart** [comment]
Can we see your current `zebra.db` ? @yur1xpp

**Comment by yur1xpp** [comment]
@PoomSmart I opened the database and this is the entry for AltDaemon:

```
Riley Testut | AltDaemon allows AltStore to install and refresh apps without a computer. | 0 | https://chariz.com/cdn/icon/swipeextenderx/icon@3x.png | com.rileytestut.altdaemon | 1708 | 1609508411 | AltDaemon | 0 | System | _var_lib_dpkg_status_ | cydia::commercial, compatible_min::ios12.0 | com.rileytestut.altdaemon-1.0-_var_lib_dpkg_status_ | 1.0 |   |   | firmware (>= 12.0), mobilesubstrate, com.creaturesurvive.libcscolorpicker | https://chariz.com/buy/swipeextenderx |   |   |   | https://altstore.io |   | Riley Testut |   |   |   |  

```
It look like Zebra did indeed fetched the wrong info, as can be seen with `https://chariz.com/cdn/icon/swipeextenderx/icon@3x.png` and `https://chariz.com/buy/swipeextenderx`. This is after Zebra were uninstalled, the folder xyz.willy.Zebra was removed, and finally Zebra is reinstalled.

**Comment by PoomSmart** [comment]
@yur1xpp Now in beta 8a you can perform "Reset Source Caches". See if that resolves your issue.

**Comment by opa334** [comment]
still happens for me in beta 8a

here's what I did:
- Updated from beta 6 to 8a
- Changes tab was empty again, I reset source cache
- Everything was okay
- I downgraded Crane from a private build I still had installed to the official version on packix
- Went into Zebra again and was greeded with this (wrong icon and wrong depiction):
![0C212F23-EC34-43B8-AF0C-43DF5133A1A4](https://user-images.githubusercontent.com/18235760/104123945-45e32280-534e-11eb-9d53-980c328a109f.jpeg)

(I wanted to post my sources here but the export feature seems to be broken too, just appears empty 
![8866D030-BF36-4651-AFB6-8D3F842C8432](https://user-images.githubusercontent.com/18235760/104124018-aa9e7d00-534e-11eb-8fb3-24ee35e9d27d.png)
)

![00C19004-3074-44A0-8F88-2C0F256D2BCB](https://user-images.githubusercontent.com/18235760/104124025-b8540280-534e-11eb-8232-5831db473b04.png)
![CB2DCEF6-480B-4085-895B-994C1EFE29BE](https://user-images.githubusercontent.com/18235760/104124029-bb4ef300-534e-11eb-8c50-d90a2f3f71bd.png)
![FAE7DA59-3206-4DA3-B207-CCE86E7FEEA0](https://user-images.githubusercontent.com/18235760/104124030-bbe78980-534e-11eb-9532-7bf307ed2958.png)
![FAA5C043-CBB2-4077-BFBB-13BC6C45A116](https://user-images.githubusercontent.com/18235760/104124033-bc802000-534e-11eb-864e-e5b88cfce284.png)
![4A5C8321-1D17-4388-8EBB-C9B73C0B39B4](https://user-images.githubusercontent.com/18235760/104124036-bdb14d00-534e-11eb-8019-2d7375c7d3f5.png)

**Comment by yur1xpp** [comment]
@PoomSmart I did exactly what @opa334 described, unfortunately this bug still present.

**Comment by wstyres** [comment]
Resolved in 2.0 alpha.
